wesnoth-ei.po

```diff
@@ -13551,161 +13518,144 @@ msgstr "Pomstím své padlé druhy!"
 
 #. [message]: speaker=Owaec
 #: data/campaigns/Eastern_Invasion/utils/final_battle/final_battle.cfg:121
-#, fuzzy
 msgid ""
 "Mal-Ravanal, at last I face you. I will have my vengeance for the "
 "destruction you wrought on my people!"
 msgstr ""
-"Mal-Ravanale, konečně ti čelím. Pomstím se za zkázu, kterou jsi způsobil "
+"Konečně ti čelím, Mal-Ravanale. Pomstím se za zkázu, kterou jsi způsobil "
 "mému lidu!"
 
 #. [message]: speaker=Dolburras
 #: data/campaigns/Eastern_Invasion/utils/final_battle/final_battle.cfg:140
-#, fuzzy
 msgid "Aye, I love the sound o’ bones crunching beneath me hammer!"
 msgstr "Jo, miluju zvuk kostí křupajících pod mým kladivem!"
 
+# Přeloženo nedoslovně a explicitněji, protože by jinak hráči nejspíš ten smysl unikl.
 #. [message]: speaker=Dolburras
 #. in regular English, "So you are the big one, aren't you? Well, the bigger they are"
 #. there is a colloquial expression, "the bigger they are, the harder they fall"
 #: data/campaigns/Eastern_Invasion/utils/final_battle/final_battle.cfg:158
-#, fuzzy
 msgid "Ooch, so you be the big ’un, aren’t ye? Well, the bigger they are..."
-msgstr "Takže ty jsi ten velký, že? No, čím větší jsou..."
+msgstr ""
+"Ooch, takže ty seš ten mocný, že? No, z velkých výšin jsou pády tvrdší..."
```

Comment:
Ještě jsem si vzpomněl, že Pavel Dobeš zpívá svou ostravštinou: "Enomže vysoko vyrostle stromy tahaju na sebe buřku a hromy...", což mi sem sedí významem, ostravštinou, kterou bych rád pro trpaslíky použil, a navíc je to český popkulturní odkaz, který by se dal uvést trpaslíkem třeba "No, jak zpiva jeden bard: Enomže..."

Jen už to teď nechci zdržovat, tak prosím dej vědět, kdyby se ti to líbilo.

Reply:
Líbí se mi to moc.